.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: dart
+sudo: false
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  - test
+  - dartfmt
+  - dartanalyzer
+
+matrix:
+  allow_failures:
+  - dart_task: dartanalyzer
```

Comment:
Do we want to just allow failures at this point and then when everything is green then protect the branch?